src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1580,6 +1581,18 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
             }
         }
 
+        if (!empty($config['globals'])) {
+            $def = $container
+                ->register('translator.globals', GlobalsTranslator::class)
```

Comment:
what about using the default `translator` service? 
I mean for twig, it is directly inside the `twig` service

Reply:
It was my first idea. What I had in mind:
* Create a new interface (in Contract ?)
* Tag services with this interface
* Call `addGlobal` with parameters given in configuration
It seems a little bit more complex.